build/release-images.sh

```diff
@@ -31,8 +31,7 @@ if [[ "${KUBE_BUILD_CONFORMANCE}" =~ [yY] ]]; then
 fi
 
 # TODO(dims): Remove this when we get rid of hyperkube image
-CMD_TARGETS="${CMD_TARGETS} cmd/kubelet cmd/kubectl"
-
+CMD_TARGETS=$(echo "${CMD_TARGETS} cmd/kubelet cmd/kubectl" | tr ' ' '\n' | sort | uniq | tr '\n' ' ')
```

Comment:
so we should put these extra targets behind if we're building the hyperkube image, IE respect `KUBE_BUILD_HYPERKUBE` in the way we do above for `KUBE_BUILD_CONFORMANCE`.

Reply:
```suggestion
if [[ "${KUBE_BUILD_HYPERKUBE}" =~ [yY] ]]; then
    CMD_TARGETS="${CMD_TARGETS} cmd/kubelet cmd/kubectl"
fi

CMD_TARGETS=$(echo "${CMD_TARGETS}" | tr ' ' '\n' | sort | uniq | tr '\n' ' ')
```